src/pages/docs/tutorials/tutorial-kitten-browser/app-setup/index.md

```diff
@@ -139,6 +142,25 @@ npm run serve
 ![Kitten Browser Step 1](KittenBrowser-Step1.png)
 > Tell the kids to avert their eyes!
 
+### ESLint Hints
+
+When you run the above command, you might see the compilation warning message below. Let's take a look at what this message means and try to resolve it briefly.
+```console
+Compiled with warnings.
+
+./src/App/App.js
+  Line 13:6:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
+
+Search for the keywords to learn more about each warning.
+To ignore, add // eslint-disable-next-line to the line before.
+```
+The `npm run serve` command (aliased as `enact serve`) will build and host your project on a web server. While building the code, it allows ESLint to perform a static analysis of the source code. ESLint helps you to find syntax errors, anti-patterns, and to write the source code in a consistent code style. See [about ESLint](https://eslint.org/docs/about/) for more information. If you only want to run a static analysis of the code you wrote, use `npm run lint` command (aliased as `enact lint`).
+
+Returning to the warning message, ESLint says img elements should have an `alt` property. That's right. When you use the `<img />` tag, it is recommended to define an `alt` property for accessibility failure to load the image. More info is available on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img). We'll add the `alt` property to the tag as follows. See if it is built successfully.
+```js
+<img src="//loremflickr.com/300/300/kitten" alt="Kitten" />
+```
+
```

Comment:
Let's move the title Conclusion here.
Change the line 135 "Conclusion" to "Serve the App" 
CC: @MikyungKim

Reply:
I'll check all links. That should be fixed..
Thank you.

Reply:
OK. "Conclusion" will be changed.